dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/logger/SdkInternalLogger.kt

```diff
@@ -253,6 +264,22 @@ internal class SdkInternalLogger(
         }
     }
 
+    private fun Map<String, Any?>?.putIfNotExistOrMapIsNull(key: String, value: Float?): MutableMap<String, Any?> {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏ Usually we prefer not to use extension functions if they're only needed in one class. It is more performant as extension function tend to create more object at the bytecode level.

Reply:
Rewritten to classic fun, thanks 🙏